src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -7284,6 +7284,18 @@ components:
         sql:
           type: string
 
+    SnapshotBuilderParentConcept:
+      type: object
+      description: >
+        ⚠️ An object describing a parent concept in the Snapshot Builder
+      properties:
+        parentId:
```

Comment:
Do we not need the name and count for rendering these on the hierarchy view? Why can't we use `SnapshotBuilderConcept` here?

Reply:
The root node is never rendered so it doesn't need these values. For any non-root node, its values are in `children`. Doing it this way avoid generating a dummy root node in the API layer, which seemed better to me.